<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAssetDialog</class>
 <widget class="QDialog" name="AddAssetDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new asset</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="3" column="1">
    <widget class="QLineEdit" name="isinEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="SymbolLbl">
     <property name="text">
      <string>Symbol:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="NameEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="TypeCombo"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="SymbolEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="NameLbl">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="TypeLbl">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="DataSrcCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="isinLbl">
     <property name="text">
      <string>ISIN</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="DataSrcLbl">
     <property name="text">
      <string>Quotes source:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>NameEdit</tabstop>
  <tabstop>TypeCombo</tabstop>
  <tabstop>isinEdit</tabstop>
  <tabstop>DataSrcCombo</tabstop>
  <tabstop>SymbolEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAssetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
